plugins/services/src/js/components/ServiceBreadcrumbs.js

```diff
@@ -121,21 +121,27 @@ class ServiceBreadcrumbs extends React.Component {
 
     const serviceStatus = service.getStatus();
     const tasksSummary = service.getTasksSummary();
-    const runningTasksCount = tasksSummary.tasksRunning;
+    const runningTasksCount = service.getTaskCount();
     const instancesCount = service.getInstancesCount();
     const isDeploying = serviceStatus === "Deploying";
 
     if (instancesCount === 0) {
       return null;
     }
 
+    let taskCountDetails = `(${runningTasksCount})`;
+
+    if (runningTasksCount !== instancesCount) {
```

Comment:
Works great @jfurrow !

What do you think about using ternary? I believe less explicit conditions keep the code cleaner. not a big deal though.
```
const taskCountDetails = runningTasksCount !== instancesCount
      ? `(${runningTasksCount} of ${instancesCount})`
      : `(${runningTasksCount})`;
```